projects/1.18/assets/create-stuff-additions/create_sa/lang/zh_cn.json

```
@@ -0,0 +1,83 @@
+{
+  "item.create_sa.vault_component": "储物组件",
+  "item.create_sa.large_filling_tank": "大型储水罐",
+  "item.create_sa.copper_pickaxe": "铜镐",
+  "item.create_sa.zinc_sword": "锌剑",
+  "item.create_sa.zinc_leggings": "锌护腿",
+  "item.create_sa.small_filling_tank": "小型储水罐",
+  "key.create_sa.increase_reach": "增加无人机的覆盖范围",
+  "item.create_sa.copper_sword": "铜剑",
+  "item.create_sa.brass_pickaxe": "黄铜镐",
+  "item.create_sa.brass_shovel": "黄铜锹",
+  "item.create_sa.brass_jetpack_chestplate": "黄铜喷气背包",
+  "item.create_sa.andesite_exoskeleton_chestplate": "安山外骨骼",
+  "item.create_sa.slime_boots": "黏液靴子",
+  "item.create_sa.copper_helmet": "铜头盔",
+  "item.create_sa.incomplete_steam_engine": "蒸汽引擎（半成品）",
+  "item.create_sa.copper_leggings": "铜护腿",
+  "item.create_sa.brass_drill_head": "黄铜钻头",
+  "item.create_sa.medium_fueling_tank": "中型燃料罐",
+  "item.create_sa.brass_hoe": "黄铜锹",
+  "item.create_sa.incomplete_hydraulic_engine": "液压引擎（半成品）",
+  "item.create_sa.medium_filling_tank": "中型储水罐",
+  "item.create_sa.copper_boots": "铜靴子",
+  "item.create_sa.flamethrower": "烈焰投掷器",
```

```suggestion
  "item.create_sa.flamethrower": "烈焰喷射器",
```
一般`Flamethrower`译作`火焰喷射器`